.travis.yml

```diff
@@ -1,19 +1,16 @@
-# Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
 os:
-  - linux
-  - osx
+    - linux
```

Comment:
It doesn't really matter but out of curiosity why change the indentation style in the default Travis file?